docs/getting-started.md

```diff
@@ -22,29 +22,30 @@ make docker-build
 MySQL setup
 -----------
 
-After you have MySQL up and running, create new empty database (/schema) & user for
+After you have MySQL up and running, create a new empty database (/schema) & user for
 `a12n-server`.
 
 ```sh
-mysql> CREATE DATABASE db_name;
-mysql> CREATE USER 'username'@'localhost' IDENTIFIED BY 'password';
+mysql> CREATE DATABASE a12nserver;
+mysql> CREATE USER 'a12nserver' IDENTIFIED BY 'password';
 ```
 
-The next step is to insert the MySQL schemas that are shipping with the git
-repository. The easiest is to just run:
+While running the application, it is possible to run into privileges issues.
```

Comment:
This is incorrect. `GRANT` is _required_. This suggests that issues _might_ happen, but it should just be part of the earlier 2 steps.